CMakeLists.txt

```diff
@@ -89,10 +89,16 @@ option(QT6 "Build with Qt6" OFF)
 if(APPLE)
   if(QT6)
     # Minimum macOS version supported by Qt 6
-    set(CMAKE_OSX_DEPLOYMENT_TARGET 10.15 CACHE STRING "Minimum macOS version the build will be able to run on" FORCE)
+    set(CMAKE_OSX_DEPLOYMENT_TARGET 10.15 CACHE STRING "Minimum macOS version the build will be able to run on")
+    if(NOT VCPKG_TARGET_TRIPLET)
+      set(VCPKG_TARGET_TRIPLET "x64-osx-min10.15")
+    endif()
   else()
     # Minimum macOS version supported by Qt 5.12
-    set(CMAKE_OSX_DEPLOYMENT_TARGET 10.12 CACHE STRING "Minimum macOS version the build will be able to run on" FORCE)
+    set(CMAKE_OSX_DEPLOYMENT_TARGET 10.12 CACHE STRING "Minimum macOS version the build will be able to run on")
```

Comment:
I have removed the `FORCE` here since it doesn't seem to be needed and using it prevents overriding it via CLI flags (which e.g. can be useful for building using newer macOS SDKs as e.g. needed for the ARM-native build)

Reply:
We may consider to join that with the above 
```
if(DEFINED ENV{VCPKG_DEFAULT_TRIPLET})
    set(VCPKG_TARGET_TRIPLET "$ENV{VCPKG_DEFAULT_TRIPLET}")
  endif()
```

can you also remove this: 
https://github.com/mixxxdj/mixxx/blob/935ba94f7e498eebd7c3710b2f0df35d989d9a7d/.github/workflows/build.yml#L64

Reply:
> We may consider to join that with the above

I thought about that but figured it would make sense to group it with the other Apple-specific configuration instead. Wdyt?

Reply:
Ok, than let's keep it here.